src/platform/site-wide/sass/consolidated-patches.scss

```diff
@@ -14,8 +14,20 @@
   padding-top: 2em;
 }
 
+.usa-grid {
+  width: $teamsite-width;
+  padding-left: 1rem;
+}
+
+.usa-grid-full {
+  width: $teamsite-width;
+}
+
 header.merger {
   z-index: 300;
+  width: 100%;
+  border: 1px solid #01133F;
+  margin: auto;
```

Comment:
These are based on #main-wrap and container styles:
<img width="969" alt="screen shot 2018-10-15 at 12 37 41 pm" src="https://user-images.githubusercontent.com/16051172/46973807-2679c400-d077-11e8-9b87-364ca48ade5a.png">